gdbinit.py

```diff
@@ -150,6 +150,12 @@ def main() -> None:
     os.environ["PWNLIB_NOTERM"] = "1"
 
     import pwndbg  # noqa: F811
+    import pwndbg.dbg.gdb
+    import pwndbg.profiling
```

Comment:
I remove the duplicate in this snippet, mostly because I want to minimize the number of modules that get imported before `pwndbg.dbg` is set to a real value.